docs/draft-ietf-rpp-requirements.clean.txt

```diff
@@ -557,12 +566,96 @@ Table of Contents
 
 18.1.  Domain Object Type
 
-18.1.1.  Internationalisation
+   *D1.1* The RPP domain object data model MUST include, at a minimum
+   the attributes defined in RFC5731: the fully qualified domain name,
```

Comment:
IIRC there has been talk about incorporating RFC5910 directly, is that covered by the "at a minimum" or should we be more explicit about things we are more sure we would add to the core protocol?